<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="textname_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #6527BE;
    color: white;
    font-size: 20px; /* Tamaño del texto */
    padding: 9px; /* Espaciado interno */
}</string>
   </property>
   <property name="text">
    <string>Ingresa el nombre</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputtext_create_cat">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit:hover {
    border: 2px solid blue;
}
QTextEdit {
    font-size: 14pt;
    font-family: Helvetica;
}


</string>
   </property>
  </widget>
  <widget class="QPushButton" name="guardar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #55d940;
    border-radius: 10px;
    color: white;
    font-size: 16px; /* Tamaño del texto */
    padding: 8px; /* Espaciado interno */
}
QPushButton:pressed {
    background-color: #A7EDE7;
}
QPushButton:hover {
   border: 1px solid blue;
}
</string>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>401</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/background.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #d94040;
    border-radius: 10px;
    color: white;
    font-size: 16px; /* Tamaño del texto */
    padding: 8px; /* Espaciado interno */
}
QPushButton:pressed {
    background-color: #A7EDE7;
}
QPushButton:hover {
   border: 1px solid blue;
}
</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="textname_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #6527BE;
    color: white;
    font-size: 20px; /* Tamaño del texto */
    padding: 9px; /* Espaciado interno */
}</string>
   </property>
   <property name="text">
    <string>de la categoría :</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>cancelar</zorder>
  <zorder>textname_2</zorder>
  <zorder>textname_1</zorder>
  <zorder>inputtext_create_cat</zorder>
  <zorder>guardar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
